app/scripts/metamask-controller.js

```diff
@@ -2105,7 +1981,6 @@ export default class MetamaskController extends EventEmitter {
           this.getAddTransactionRequest({ transactionParams, dappRequest }),
         ),
       // msg signing
-      ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)
```

Comment:
This type of build fence essentially just excludes code from MMI builds. This type of code fence has also been completely removed here, since it only impacts the MMI build that we're going to remove in the next PR.